manta-accounting/src/wallet/mod.rs

```diff
@@ -339,7 +339,7 @@ where
             let ReadResponse {
                 should_continue,
                 data,
-            } = self.read_from_ledger().await?;
+            } = self.read_from_ledger(&checkpoint).await?;
```

Comment:
the checkpoint here at first will be Default(0), so every time the wallet trying to call `initial_sync`, it will sync from beginning? So I suppose this initial_sync is only called once when user first time install the wallet. What if usre disable the wallet and enable it again, do we call initial_sync again? @DanielZhangReal

Reply:
It will only be called once, and the result will be saved to indexedDB after success. 
Disabling and enabling will not affect indexedDB. 
Also, this logic only exists when the user creates a wallet, not the import logic